versioned_docs/version-2.0.0/server/sdk-installation/go.md

```diff
@@ -22,6 +22,45 @@ import WhatAreKeployFeatures from './index.md'
 
 <WhatAreKeployFeatures/>
 
+## 🛠️ Language Specific Requirements
+
+| Programming Language | Prerequisites                                                                                                                                                                                                                                              |
+| :------------------: | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|          go          | 1. The application should have a graceful shutdown to stop the API server on `SIGTERM` or `SIGINT` signals. Refer [appendix](#appendix) for basic implementation of graceful shutdown function. <br/> 2. The go binary should be built with `-cover` flag. |
+
+## Graceful Shutdown
+
+It is important that the application is shutdown gracefully. In case of Golang, function for graceful shutdown:
+
+```go
+func GracefulShutdown() {
+	stopper := make(chan os.Signal, 1)
+	// listens for interrupt and SIGTERM signal
+	signal.Notify(stopper, os.Interrupt, os.Kill, syscall.SIGKILL, syscall.SIGTERM)
+	go func() {
+		select {
+		case <-stopper:
+			os.Exit(0)
+		}
+	}()
+}
+
+func main() {
+
+	port := "8080"
+
+	r := gin.Default()
+
+	r.GET("/:param", getURL)
+	r.POST("/url", putURL)
+	// should be called before starting the API server from main()
+	GracefulShutdown()
+
+	r.Run()
+}
+```
+
+
 ## Usage
 
```

Comment:
seems like the note of jvm would still remain in this.

Reply:
@shivamsouravjha  I'm not quite sure which note you're referring to, as each language section appears to be dedicated solely to its respective language. Can you please guide me to the actual issue?

Reply:
<img width="1438" alt="image" src="https://github.com/user-attachments/assets/71795c0d-924b-4f98-aa82-e2fa62752796">
here this is a go code still it has java code.

In java there is go code 
<img width="1339" alt="image" src="https://github.com/user-attachments/assets/212603ae-a5a8-485f-91f3-c72152d48992">

In js it has go and java code
![Uploading image.png…]()

<img width="1435" alt="image" src="https://github.com/user-attachments/assets/f75eb5ac-7b1f-4806-89de-b0980f2d09c3">

Reply:
I have made all the changes that you are referring to here, Wait let me share what my local build contains.

# Go
![image](https://github.com/user-attachments/assets/8d655d80-ce87-4c87-ab3a-7cbaa758c6f4)

# Java
![image](https://github.com/user-attachments/assets/39dd49ab-13e5-4af4-9d86-ef980594233d)

# Javascript
![image](https://github.com/user-attachments/assets/680cfcff-7f60-4b3d-9edd-d796d5d001ef)

# Python
![image](https://github.com/user-attachments/assets/3856ae00-99d5-4257-bb38-48fa3831dfbe)